src/ImageRunModal.jsx

```diff
@@ -1089,6 +1165,24 @@
                 </Tabs>
             </Form>
         );
+
+        const createRunText = this.props.prefill ? _("Clone and run") : _("Create and run");
+        const createText = this.props.prefill ? _("Clone") : _("Create");
+
+        const runModalFooter = (
+            <>
+                <Button variant='primary' id="create-image-create-run-btn" onClick={() => this.onCreateClicked(true)} isDisabled={(!image && selectedImage === "")}>
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1270-20230428-110256-01fd85db-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)